src/nft/utils/address.ts

```diff
@@ -0,0 +1,31 @@
+import { getAddress } from 'ethers/lib/utils'
+
+/**
+ * Check an Ethereum address for validity
+ * @param address blockchain address
+ * @returns original address or "false"
+ */
+export function isAddress(value: string): string | false {
```

Comment:
This is already done by isAddress from `@ethersproject/address`, and should not be duplicated.